cmd/buildah/mount.go

```diff
@@ -21,9 +27,15 @@ func init() {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return mountCmd(cmd, args, noTruncate)
 		},
-		Example: `buildah mount
+		Example: `
+  buildah mount
```

Comment:
This line start just after the back tick in the line above, as it had previously.

Reply:
Fixed